Automattic/wp-calypso
# Help Center: update rating logic for improved clarity

Related to https://github.com/Automattic/wp-calypso/issues/96846

## Proposed Changes

* Updates the message rating logic

## Why are these changes being made?


* Fix rating logic functionality.

## Testing Instructions



* Checkout branch or use live link
* Navigate to `/home?flags=help-center-experience`
* Open the Help Center, start a new conversation with AI, send a message, click on the thumbs down / thumps up button. Test the functionality.
* Navigate away from the chat and return to verify the thumbs down/thumbs up feedback button do not appear.
* Repeat these steps with a HE chat.
* Repeat the steps without the flag enabled.

## Pre-merge Checklist
